_posts/2016-12-06-deprecating-nodes.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Deprecating Nodes
+author: David Davis
+tags:
+  - release
+  - platform
+  - 3.0
+---
+
+Nodes were introduced in Pulp in 2.1.0 and provided a way to keep multiple Pulp
+servers in sync. However, maintaining Nodes in Pulp meant having to maintain a
+separate codebase and given that much of the functionality of Nodes can be
+acheived by running Pulp straight out of the box, we've decided to remove Nodes
+in our next major release of Pulp 3.0. With Pulp 2.11, Nodes will be officially
+deprecated.
+
+In this blog post, I want to outline how users can sync content across Pulp
+servers without using Nodes and how tools like cron and systemd can be used to
+keep content updated across Pulp servers. I am going to illustrate a two-server
+example in which I'll call one Pulp server the parent and the other the child.
+However, this distinction is arbitrary as both servers are configured the same
```

Comment:
s/configured the same way/each a standard Pulp server installation without any Nodes packages installed/.